llvm/lib/Transforms/Vectorize/VectorCombine.cpp

```diff
@@ -1742,6 +1742,10 @@ static Value *generateNewInstTree(ArrayRef<InstLane> Item, FixedVectorType *Ty,
   if (auto *BI = dyn_cast<BinaryOperator>(I))
     return Builder.CreateBinOp((Instruction::BinaryOps)BI->getOpcode(), Ops[0],
                                Ops[1]);
+  if (auto CI = dyn_cast<CmpInst>(I))
+    return Builder.CreateCmp(CI->getPredicate(), Ops[0], Ops[1]);
+  if (auto SI = dyn_cast<SelectInst>(I))
```

Comment:
auto *SI

Reply:
In the future, to catch such nits, I would recommend getting the official clangd extension from LLVM in VSCode (if you use VSCode to contribute to LLVM): it will automatically yellow-underline such instances, and prompt you about the fix.

Reply:
It'd be nice for the CI code_formatter stage to pick these up